pkg/controller/static-resource-controller/controller.go

```diff
@@ -99,6 +99,8 @@ type reconcilerStatus struct {
 // +kubebuilder:rbac:groups=apps,resources=statefulsets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=coordination.k8s.io,resources=leases,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cert-manager.io,resources=certificaterequests,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cert-manager.io,resources=issuer,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Rbac for `issuer` is not required, since we're not enabling `upstreamAuthority.certManager.ca.*`.

More background: upstream introduced it because they check the CA issuer's readiness in the initContainer https://github.com/spiffe/helm-charts/commit/b3459445d036523a1c8d3fa15909c57dd9d6d9c4